tests/QueryBuilderTest.php

```diff
@@ -737,4 +737,25 @@ public function testValue()
         $this->assertEquals('Herman', DB::collection('books')->value('author.first_name'));
         $this->assertEquals('Melville', DB::collection('books')->value('author.last_name'));
     }
+
+    public function testHintOptions()
+    {
+        DB::collection('items')->insert([
+            ['name' => 'fork',  'tags' => ['sharp', 'pointy']],
+            ['name' => 'spork', 'tags' => ['sharp', 'pointy', 'round', 'bowl']],
+            ['name' => 'spoon', 'tags' => ['round', 'bowl']],
+        ]);
+
+        $results = DB::collection('items')->hint(['$natural' => -1])->get();
+
+        $this->assertArraySubset(['name' => 'spoon'], $results[0]);
```

Comment:
function `assertArraySubset` is deprecated and will be removed in PHPUnit 9
https://packagist.org/packages/dms/phpunit-arraysubset-asserts maybe try to use this package

Reply:
I've changed `assertArraySubset` to `assertEquals` and changed test as well.

We can't rely on 3rd party because it's lacking a support for phpunit 8.4  https://github.com/rdohms/phpunit-arraysubset-asserts/pull/12